vtexdocs/devportal
# Feat/cookies

#### What is the purpose of this pull request?



#### What problem is this solving?

To add cookie consent.

#### How should this be manually tested?

Open the preview and check if the cookie consent bar appears in the bottom of the screen and check, before giving consent, if the only cookies stored are next related cookies. After consenting to the use of cookies you should be able to see cookies with google.com as a domain and the cookie _ALGOLIA.

More details on the task and cookies can be found [here](https://www.notion.so/vtexhandbook/Componente-de-uso-de-cookies-70c5bb07f76d480a9c4ba07ead0efcbf?pvs=4#d534d47f46584a1ab3c8ea3d595c5eb8).

#### Screenshots or example usage

#### Types of changes
